scaletest/scaletest.sh

```diff
@@ -151,7 +156,21 @@ else
 fi
 KUBECONFIG="${PROJECT_ROOT}/scaletest/.coderv2/${SCALETEST_NAME}-cluster.kubeconfig"
 echo "Waiting for Coder deployment at ${SCALETEST_CODER_URL} to become ready"
-maybedryrun "$DRY_RUN" kubectl --kubeconfig="${KUBECONFIG}" -n "coder-${SCALETEST_NAME}" rollout status deployment/coder
+max_attempts=10
+for attempt in $(seq 1 $max_attempts); do
+	maybedryrun "$DRY_RUN" curl --silent --fail --output /dev/null "${SCALETEST_CODER_URL}/api/v2/buildinfo"
+	curl_status=$?
+	if [[ $curl_status -eq 0 ]]; then
+		break
+	fi
+	if attempt -eq $max_attempts; then
+		echo
+		echo "Coder deployment failed to become ready in time!"
+		exit 1
+	fi
+	echo "Coder deployment not ready yet (${attempt}/${max_attempts}), sleeping 3 seconds"
+	maybedryrun "$DRY_RUN" sleep 3
+done
```

Comment:
review: there is a race condition between the `rollout status` returning true and the service actually becoming ready; so I'm just going back to curl :-)